docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -223,6 +235,70 @@ The nodes should be configured to allow key based SSH connections as user ``root
 
    admin/run-acceptance-tests --distribution centos-7 --provider managed --config-file config.yml
 
+If you are using the ``managed`` provider with ``AWS`` nodes, you  will need to supply both the private and public IP addresses for each node.
+AWS nodes do not have public IP addresses configured in the operating system; instead Amazon routes public IP traffic using NAT.
+In this case the acceptance tests need a hint in order to map the private IP address reported by the Flocker ``/state/nodes`` API to the public node address.
```

Comment:
There's 
 * https://clusterhq.atlassian.net/browse/FLOC-2137 and
 * https://github.com/ClusterHQ/flocker/compare/public-ips-FLOC-2137 

...where @tomprince introduced a new node hostname item to `agent.yml`.